Remora.Discord.Commands/Extensions/ApplicationCommandDataExtensions.cs

```diff
@@ -50,6 +50,32 @@ public static void UnpackInteraction
         out IReadOnlyDictionary<string, IReadOnlyList<string>> parameters
     )
     {
+        if (commandData.Type is ApplicationCommandType.User or ApplicationCommandType.Message)
+        {
+            var parameterName = commandData.Type switch
+            {
+                ApplicationCommandType.Message => "message",
+                ApplicationCommandType.User => "user",
+                _ => throw new InvalidOperationException()
```

Comment:
This line should never be reached.

Reply:
I would leave it there to avoid oversight when for example in the future there might be additional command types.
That might have some special handling.

Reply:
Yes, having a throwing default is good practice and makes for safer code. Keep it as it is.